packages/mdc-grid-list/README.md

```diff
@@ -0,0 +1,195 @@
+# MDC Grid List
+
+MDC Grid List provides a RTL-aware Material Design Grid Lists component adhering to the
+[Material Design Grid List spec](https://material.io/guidelines/components/grid-lists.html).
+Grid List is best suited to presenting homogenous data, typically images. Each data in
+grid list is called a tile and tile maintains fixed width, height and padding between each
+other as window resizes. Meanwhile, margin changes to keep grid list remain centered.
+
+
+## Installation
+
+```
+npm install --save @material/grid-list
+```
+
+
+## Usage
+
+Basic grid list has the following structure:
+
+```html
+<div class="mdc-grid-list">
+  <ul class="mdc-grid-list__tiles">
+    <li class="mdc-grid-tile">
+      <div class="mdc-grid-tile__primary">
+        <img class="mdc-grid-tile__primary-content" src="images/1-1.jpg" />
```

Comment:
SGTM. But could we keep the thing in the demo using `images/1-1.jpg`? I tried other resources but they are not as beautiful as the orange and blue tiles do.